client/package.json

```diff
@@ -27,7 +27,7 @@
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/bazaarvoice/clientresponse-demo"
+    "url": "https://github.com/bazaarvoice/response-demo"
   },
   "team": {
     "name": "API Infrastructure",
```

Comment:
Could you change the e-mail address here as well. I am not able to do an in-line suggestion here